src/components/Blogs/index.test.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import Blogs from "./index";
+
+global.fetch = jest.fn(() =>
+  Promise.resolve({
+    json: () =>
+      Promise.resolve({
+        title: {
+          rendered: "Women standing together",
+        },
+      }),
+  })
+);
+
+describe("Blogs", () => {
+  it("renders correctly and matches the snapshot", () => {
+    const tree = renderer.create(<Blogs />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it("renders title", async () => {
+    Blogs.fetchBlogPosts = jest.fn();
```

Comment:
this is what your test is doing

1. create a random new function called Blogs.fetchBlogPosts that does nothing (this is not the same as the function in your thing)
2. call this function. console.log title here. it's undefined
3. compare this "undefined" title to a snapshot that also says "undefined."